src/test/kotlin/minesweeper/domain/LandTest.kt

```diff
@@ -0,0 +1,38 @@
+package minesweeper.domain
+
+import minesweeper.domain.tile.Tiles
+import minesweeper.domain.tile.pos.Coordinate
+import minesweeper.domain.tile.state.set.Mine
+import minesweeper.domain.tile.state.set.NotChecked
+import org.junit.jupiter.api.Test
+
+class LandTest {
+    @Test
+    fun `Land - 문자열 타일 목록 반환 테스트`() {
+        // given
+        val tiles = Tiles(
+            arrayListOf(
+                Mine(Coordinate.of(0, 0)), NotChecked(Coordinate.of(0, 1), false),
+                NotChecked(Coordinate.of(1, 0), false), Mine(Coordinate.of(1, 1))
+            )
+        )
+        val land = Land.of(FIXED_CALIBRATED_POSITION, FIXED_CALIBRATED_POSITION, tiles)
+
+        // when
+        val actual = land.getTiles()
+
+        // then
+        val expected = listOf(
+            listOf("*", "C"),
+            listOf("C", "*")
+        )
+
+        assert(actual == expected)
```

Comment:
```suggestion
        assertThat(actual).isEqualTo(expected)
```
assertThat을 활용하는걸 권장드립니다!

참고: https://jongmin92.github.io/2020/03/31/Java/use-assertthat/

Reply:
세심한 부분까지 확인해 주셔서 감사합니다 🙏 특히 전달해 주신 참고자료가 많은 도움이 되었습니다! 추후에도 테스트 코드 작성시에 `assertThat`을 활용하겠습니다 😄 아래와 같이 수정 반영하였습니다! 😃 

- https://github.com/next-step/kotlin-minesweeper/commit/1ea3b54b4436131da4b91d186c9115b51efae879